uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-request.html

```diff
@@ -0,0 +1,177 @@
+                  <i class="fa fa-asterisk"></i>
+                </label>
+                <div class="col-xs-4">
+                  <div class="input-group chr-datepicker">
+                    <input type="text"
+                           class="form-control"
+                           uib-datepicker-popup
+                           show-button-bar="false"
+                           datepicker-options="{startingDay: 1,showWeeks: false}"
+                           ng-model="modal.leaveRequest.fromDate"
+                           is-open="modal.leaveRequest.showDatePickerFrom"/>
+                    <span class="input-group-btn">
+                        <button type="button" class="btn btn-default fa fa-calendar"
+                                ng-click="modal.leaveRequest.showDatePickerFrom = true"></button>
+                      </span>
+                  </div>
+                </div>
+                <div class="col-xs-4">
+                  <div class="chr_custom-select chr_custom-select--full">
+                    <select>
+                      <option value="1" selected>1/2 - PM</option>
+                      <option value="2">1/2 - AM</option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-xs-12 form-horizontal">
+              <div class="form-group">
+                <label class="col-xs-4 control-label">To Date
+                  <i class="fa fa-asterisk"></i>
+                </label>
+                <div class="col-xs-4">
+                  <div class="input-group chr-datepicker">
+                    <input type="text"
+                           class="form-control"
+                           uib-datepicker-popup
+                           datepicker-options="{startingDay: 1,showWeeks: false}"
+                           show-button-bar="false"
+                           ng-model="modal.leaveRequest.toDate"
+                           is-open="modal.leaveRequest.showDatePickerTo"/>
+                    <span class="input-group-btn">
+                        <button type="button" class="btn btn-default fa fa-calendar"
+                                ng-click="modal.leaveRequest.showDatePickerTo = true"></button>
+                      </span>
+                  </div>
+                </div>
+                <div class="col-xs-4">
+                  <div class="chr_custom-select chr_custom-select--full">
+                    <select>
+                      <option value="1" selected>1/2 - PM</option>
+                      <option value="2">1/2 - AM</option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <hr/>
+          <div class="row form-group">
+            <div class="col-xs-4">Opening Balance</div>
```

Comment:
This is actually, semantically, a definition list
```html
<dl>
  <dt>Opening Balance</dt>
  <dd>24.0</dd>
</dl>
```
but i'll leave it to you to see how easy it is to style